### DISCUSSION TITLE
[bug]

### DISCUSSION THREAD

**Discussion by staniuuu** [YetiForceCompany/YetiForceCRM#15546] [General]
Hi,

I import leads from .csv file and my lead in CRM just disappear. I have only this leads from .csv.
It's a bug! How Can I back my leads before?

**Comment by staniuuu** [comment]
any help?

**Reply by rskrzypczak** [reply]
I do not see a relationship between the elements you mentioned. The import functionality does not delete existing records.
Nobody will be able to help you based on the information you provided.
However, for the sake of clarity:
![image](https://user-images.githubusercontent.com/10221999/146214437-abbc57ab-7cf0-4fa6-b4e3-8179451bc4b5.png)

**Reply by staniuuu** [reply]
I don't have this option

**Reply by mariuszkrzaczkowski** [reply]
There is no this option in your version, check if you have a recycle bin in the menu

**Comment by staniuuu** [comment]
I don't have this option 
![image](https://user-images.githubusercontent.com/94536532/146733119-1e6b02f3-3528-4e33-9dff-8eb5b388037e.png)

**Reply by mariuszkrzaczkowski** [reply]
And what version of CRM do you have, I can see that it is older?

**Reply by staniuuu** [reply]
I have ver. 3.4.0

**Comment by staniuuu** [comment]
any help?

**Reply by bpabiszczak** [reply]
We can help but only if you are using the latest version of the system.

**Reply by staniuuu** [reply]
How Can I update ?

**Reply by rskrzypczak** [reply]
https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/system-update

**Reply by staniuuu** [reply]
Czy istnieje jakiś szybki sposób aby aktualizować z wersji 3.2 do najnowszej? Czy musimy aktualizować każdą paczką co 0.1 wersji?

**Reply by bpabiszczak** [reply]
> Czy istnieje jakiś szybki sposób aby aktualizować z wersji 3.2 do najnowszej? Czy musimy aktualizować każdą paczką co 0.1 wersji?

Należy aktualizować po kolei.
